inference-contract/docs/notebooks/templates/token-issuer.py

```diff
@@ -14,26 +14,24 @@
 
 # %% [markdown]
 # # Token Issuer Notebook
-# This section enables customization of the token that will be minted. Edit the variables in the section below as necessary. 
+# This section enables customization of the token that will be minted. Edit the variables in the section below as necessary.
+#
+# * token_owner : the identity of the token creator
+# * token_class : the name of tokens that will be generated, this is only used to simplify local access (e.g. context file name)
+# * token_description : a description of the asset associated with the minted tokens
+# * token_metadata : additional information about the token
+# * count : the number of tokens to mint for the asset
+# *service_host : the host for the eservice where tokens will be minted, this will use the default service group
 #
-# identity : the identity of the token creator
-# service_host : the host for the eservice where tokens will be minted, this will use the default service group
-# token_class : the name of tokens that will be generated, this is only used to simplify local access (e.g. context file name)
-# token_description : a description of the asset associated with the minted tokens
-# token_metadata : additional information about the token
-# count : the number of tokens to mint for the asset
-
 # Note that the notebook assumes that there is a key file for the identity of the form: `${keys}/${identity}_private.pem`.
 
 
 # %% tags=["parameters"]
-token_owner = 'user1'
-token_class = 'mytoken'
 token_description = 'this is my token'
-token_metadata = 'created by {}'.format(token_owner)
+token_metadata = 'additional token metadata'
 count = 1
-service_host = 'localhost'
-instance_identifier = ''
+# rest of the parameters such as token_owner, token_class, service_host, and instance_identifier will get auto filled from factory via papermill
+# note that papermill places the parameters cell after first paramters cell in this notebook
 
```

Comment:
have you verified that this works? papermill generally requires that the template contain complete list of parameters. it isn't pretty, but if you're setting through the templating mechanism it has to be there.

Reply:
yes, tested, paperill does not need repeating all parameters. per papermill documentation, papermill populates the instance notebook with parameters after the first cell in the target notebook that has parameters. (that's why we currently see new parameters appearing after the default set). there is no need for the default set to have anything that papaermill will add. Yes, agreed, warnings will be generated while running the factory notebook.

Reply:
it actually doesn't work. an error is generated. lets just add the variables that are necessary.

Reply:
thanks, added back all parameters that are being parametrized by papermill.